README.md

```diff
@@ -8,30 +8,25 @@ Building
 
 For this crate to build, `libmysqlclient` must be installed on your system
 (`brew install mysql` on macOS, `apt-get install libmysqlclient-dev` on Ubuntu,
-included with the server distribution on Windows). Additionally, either
-`pkg-config` or `mysql_config` must be present and able to successfully locate
-`libmysqlclient`.
+included with the [MySQL Server distribution](https://dev.mysql.com/downloads/mysql/)
+on Windows).
 
 The build script of the crate will attempt to find the lib path of
 libmysql-client using the following methods:
 
-- First, it will attempt to use pkg-config to locate it. All the config options,
+- First, it will attempt to use `pkg-config` to locate it. All the config options,
   such as `PKG_CONFIG_ALLOW_CROSS`, `PKG_CONFIG_ALL_STATIC` etc., of the crate
   [pkg-config](http://alexcrichton.com/pkg-config-rs/pkg_config/index.html)
   apply.
 - MSVC ABI builds will then check for a [Vcpkg](https://github.com/Microsoft/vcpkg)
   installation using the [vcpkg cargo build helper](https://github.com/mcgoo/vcpkg-rs).
   Set the `VCKPG_ROOT` environment variable to point to your Vcpkg installation and
   run `vcpkg install libmysql:x64-windows` to install the required libraries.
+- Next, the environment variable `MYSQLCLIENT_LIB_DIR` will be checked. This should
+  contain the path to the mysql `lib` folder. On windows this will be similar to
+  `C:\Program Files\MySQL\MySQL Connector.C 6.1\lib\vs14`.
```

Comment:
I don't think we should give a full path here. Whether `vs14` is used will depend on the version of visual studio you're using, and this can change dramatically based on how the user installed MySQL

Reply:
I disagree: it's much easier as a user to see a full path and translate that into the equivalent path on one's own system than it is to figure it out based on a description, or worse, nothing. How about if I clarify that this path will differ based on your visual studio version and mysql installation location?